src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1015.UriMustBeAbsolute_Invalid.sarif

```diff
@@ -22,6 +22,25 @@
                 }
               },
               "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
```

Comment:
>helpUri [](start = 15, length = 7)

We want every test file to trigger exactly one kind of error. Apparently, the test files for SARIF1015 use an old schema version, so the output from that test now includes SARIF1020 as well. To avoid this, update the `$schema` property in the SARIF1015 test files.